.github/workflows/pull-request.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       matrix:
         include:
           - target_option: ""
-          - target_option: "--target=wasm32-unknown-unknown"
+          - target_option: "--exclude=prqlc --target=wasm32-unknown-unknown"
```

Comment:
Better would be to promote this setting to the crate, so that `cargo build --target=wasm32-unknown-unknown` works on a fresh clone.

I'm not sure whether there's an easy way of doing that. We can do what we used to do — have the whole thing behind an attribute, so it compiles almost nothing with `wasm` (just a main that says "This doesn't work on `wasm`"!)

(I can take these btw)

Reply:
Yes, please do take this :D